### PR TITLE
Updated README.md to align with latest build

### PR DESCRIPTION
**Pull request by dgraney**
* Changed how you get a summoner
* Added a deprecated tag to the summoner.GetRankedStats() blurb
* Created a blurb on how to get champion masteries as a means of displaying the interaction between regular and static api. (Not sure if this is necessary, but thought I may as well add ¯\_(ツ)_/¯ ).

Note: for some reason it isn't seeing the change to StatsStatic.cs from before :/

### CODE REVIEW

File: RiotSharp/StaticDataEndpoint/StatsStatic.cs [hunk 1]
```diff
@@ -8,135 +8,330 @@ namespace RiotSharp.StaticDataEndpoint
     public class StatsStatic
     {
         internal StatsStatic() { }
-
+        
+        /// <summary>
+        /// Flat Armor value for the object.
+        /// </summary>
```
Comment (BenFradet) [inline RiotSharp/StaticDataEndpoint/StatsStatic.cs]: could you remove this commit as it's already been merged?

Comment (BenFradet) [reply RiotSharp/StaticDataEndpoint/StatsStatic.cs]: You should be able to do so with a `git rebase origin/develop` if `origin` is the name of this remote.

File: README.md [hunk 2]
```diff
@@ -47,15 +47,15 @@ To get a summoner:
...
+  var summoner = api.GetSummonerByName(Region.euw, "StopOFlop");
 }
 catch (RiotSharpException ex)
 {
   // Handle the exception however you want.
 }
 ```
 
-To get the stats in ranked for a specific champion for this summoner:
+**(DEPRECATED)** To get the stats in ranked for a specific champion for this summoner:
```
Comment (BenFradet) [inline README.md]: I think we can remove the whole paragraph since people won't be able to do it.

**Comment by dgraney** [comment]
Will make these changes, it may be a bit. A tad busy

File: RiotSharp/StaticDataEndpoint/StatsStatic.cs (hunk 1 above)
Comment (dgraney) [reply RiotSharp/StaticDataEndpoint/StatsStatic.cs]: You'll have to excuse my lack of knowledge, but I used that command to no avail. the remote is named origin. Any advice?

Comment (dgraney) [reply RiotSharp/StaticDataEndpoint/StatsStatic.cs]: Probably causing more problems than I'm solving :)

**Comment by dgraney** [comment]
Well at the very least, it seems that the commit still exists... however it isn't saying it is a changed file. I'm not sure if that's sufficient. Also deleted that blurb on the ranked stats.

**Comment by dgraney** [comment]
awesome, thanks for the heads up

**Comment by BenFradet** [comment]
Usually it's good practice to have a different branch for each pull request, the flow boils down to:

```bash
git checkout develop
git pull upstream develop # to have a up-to-date copy of develop
git checkout -b my-fix       # creates a new branch dedicated to the fix
...
```

**Pull request merged**